### PR TITLE
Connect attendee profile to backend

### PR DESCRIPTION
**Pull request by ruioliveira02**

### CODE REVIEW

File: pages/attendee/profile.js [hunk 1]
```diff
@@ -31,9 +55,19 @@ function Profile() {
...
+                onClick={() => {
+                  if (editing) {
+                    editUser(username);
+                    setEditing(false);
+                  } else {
+                    setEditing(true);
+                  }
+                }}
+              >
+                {editing ? "Save Changes" : "Edit"}
```
Comment (nelsonmestevao) [inline pages/attendee/profile.js:70]: It looks broken :/ 

image

File: pages/attendee/profile.js [hunk 2]
```diff
@@ -84,60 +109,85 @@ function Profile() {
 
             <div className="grid-cols-2 overflow-hidden">
               <div className="col-span-1 float-left w-1/2 font-iregular">
-                💰170 Tokens
+                💰{user.token_balance} Tokens
               </div>
               <div className="col-span-1 float-left w-1/2 font-iregular">
-                🏅68 Badges
+                🏅{user.badge_count} Badges
```
Comment (nelsonmestevao) [inline pages/attendee/profile.js:115]: Maybe add a space between the emoji and the text, what do you think? 
```suggestion
                🏅 {user.badge_count} Badges
```

File: pages/attendee/profile.js [hunk 3]
```diff
@@ -84,60 +109,85 @@ function Profile() {
...
-              CV!
+              Get a chance to win a spot at the Corporate dinner by submiting
+              you CV!
             </p>
 
-            <div className="sm:grid sm:grid-cols-3 sm:items-start sm:gap-4 sm:pt-5">
+            <a
+              href="mailto:cv@seium.org"
+              className="inline-block h-auto pl-6 pb-5 text-quinary underline"
+            >
```
Comment (nelsonmestevao) [inline pages/attendee/profile.js:177]: Use a blue button like the ones we use everywhere :pray:

File: components/Auth/AuthProvider.js [hunk 4]
```diff
@@ -102,13 +102,13 @@ export function AuthProvider({ children }) {
...
+  function editUser(nickname) {
     setLoading(true);
 
     api
-      .editUser(values)
-      .then((user) => {
-        setUser((oldUser) => ({ ...oldUser, ...user }));
+      .editUser(user.id, nickname)
+      .then((at) => {
+        setUser((oldUser) => ({ ...oldUser, attendee: at }));
```
Comment (nelsonmestevao) [inline components/Auth/AuthProvider.js:111]:
```suggestion
        setUser((oldUser) => ({ ...oldUser, ...at }));
```

**Review by miguelbrandao — APPROVED** [review]

**Pull request merged**